docs/adding-languages.md

```diff
@@ -23,6 +23,7 @@ Add an SVG logo into the `public/languages/` directory.
 1. Default the default code for the new language (such as `defaultJsonCode` or `defaultCssCode`).
 1. Add an entry in the `defaultCode` variable for the new language's default code.
 1. Export a variable containing the default options for the new language (such as `defaultJsonOptions` or `defaultCssOptions`).
+1. Add an entry to the `esquerySelectorPlaceholder` constant for the new language, using an appropriate example selector.
```

Comment:
It would be great if we could actually link to where exactly this is in code? That would be better for someone newly exploring the repo? WDYT